src/app/public/modules/image/image.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
```

Comment:
Import statements need to follow code style guidelines.
See: https://docs.blackbaud.com/skyux-team-docs/style#imports